source/addins/DistanceAndDirectionLibrary/Helpers/DebounceDispatcher.cs

```diff
@@ -39,7 +41,7 @@ public class DebounceDispatcher
             Dispatcher disp = null)
         {
             // kill pending timer and pending ticks
-            timer?.Stop();
+            timer.Stop();
```

Comment:
you need to null check here before using timer.Stop, that's what the ?. does.  Null Propagation.

Reply:
Added in https://github.com/Esri/distance-direction-addin-dotnet/pull/640/commits/b9a03167923aedcff3caa85e8c2c24a98a09b609